--- compat.h	2022-06-27 00:00:00.000000000 +0000
+++ compat.h	2026-02-09 00:00:00.000000000 +0000
@@ -36,10 +36,6 @@
 #error "WireGuard requires Linux >= 3.10"
 #endif

-#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 6, 0)
-#error "WireGuard has been merged into Linux >= 5.6 and therefore this compatibility module is no longer required."
-#endif
-
 #if defined(ISRHEL7)
 #include <linux/skbuff.h>
 #define headers_end headers_start
